lib/dom-parser.js

```diff
@@ -101,7 +101,10 @@ function normalizeLineEndings(input) {
  * @see https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-parsing-and-serialization
  */
 function DOMParser(options) {
-	options = options || { locator: true };
+	options = options || {};
+	if (typeof options.locator === 'undefined') {
```

Comment:
How is this different from
```suggestion
	if (options.locator === undefined) {
```
?

Reply:
In this case it doesn't make any difference, so yeah I'll change that